Demo/src/androidTest/java/com/braintreepayments/demo/test/DropInTest.java

```diff
@@ -216,6 +220,31 @@ public void exitsAfterCancelingAddingAPaymentMethod() {
         onDevice(withText("Add Payment Method")).check(text(equalToIgnoringCase("Add Payment Method")));
     }
 
+    @Test(timeout = 60000)
+    public void deletesPaymentMethod() {
```

Comment:
Do we need to do any verification that the customer we choose does not have existing payment methods? Or do we ensure that we clean up after ourselves somewhere else?

Reply:
I think I can play around with more assumptions to verify that we're in the right state for this test.